packages/account/tsconfig.json

```diff
@@ -2,7 +2,6 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "./dist",
-        "rootDir": "./src",
```

Comment:
Why are we removing `rootDir` ? and setting base url as `./` ?

Reply:
@jim-deriv could u please add your explanation here too